src/uu/sort/src/sort.rs

```diff
@@ -146,7 +148,9 @@ enum SortError {
     CompressProgTerminatedAbnormally {
         prog: String,
     },
-    TmpDirCreationFailed,
+    TmpDirCreationFailed {
```

Comment:
To reflect the change in the error message I think it would make sense to rename this variant to something like `TmpFileCreationFailed`.